client/state/posts/test/selectors.js

```diff
@@ -361,4 +362,44 @@ describe( 'selectors', () => {
 			] );
 		} );
 	} );
+
+	describe( '#isRequestingSitePost()', () => {
+		it( 'should return false if no request has been made', () => {
+			const isRequesting = isRequestingSitePost( {
+				posts: {
+					siteRequests: {}
+				}
+			}, 2916284, 841 );
+
+			expect( isRequesting ).to.be.false;
+		} );
+
+		it( 'should return true if a request is in progress', () => {
+			const isRequesting = isRequestingSitePost( {
+				posts: {
+					siteRequests: {
+						2916284: {
+							841: true
+						}
+					}
+				}
+			}, 2916284, 841 );
+
+			expect( isRequesting ).to.be.true;
+		} );
+
+		it( 'should return false if a request has finished', () => {
+			const isRequesting = isRequestingSitePost( {
+				posts: {
+					siteRequests: {
+						2916284: {
+							841: false
+						}
+					}
+				}
+			}, 2916284, 841 );
+
+			expect( isRequesting ).to.be.false;
+		} );
```

Comment:
>might want a test where posts.siteRequests doesn't exist

`siteRequests` is never unset, per https://github.com/Automattic/wp-calypso/pull/3108#discussion_r52023383